defenders.js

```diff
@@ -142,6 +147,12 @@ class Defender {
       this.timer++;
     }
   }
+
+  //Function to destroy the defender
+  sell(){
+    defenders.splice(defenders.findIndex((item)=>item==this)); // Removes this defender from the defenders array
```

Comment:
Seeing as though we already know the index of the defender that needs to be deleted we can pass that as a parameter into `Sell()` and splice it that way.

Reply:
As a side note, can you please comment on an issue if you're going to be working on it to avoid two people working on the same thing?